### PR TITLE
Adds teleporter beacon to Canary

### PR DESCRIPTION
**Pull request by Snowy1237**
Adds a teleporter beacon to the Canary.

I have no idea why the propulsion sensors had a space added, hopefully that does not break them.

### CODE REVIEW

**Comment by Snowy1237** [comment]
!review

**Comment by Snowy1237** [comment]
!review

File: maps/sccv_horizon/sccv_horizon-1_deck_1.dmm [hunk 1]
```diff
@@ -12385,7 +12385,7 @@
 	frequency = 1442;
 	input_tag = "port_prop_in";
 	output_tag = "port_prop_out";
-	sensors = list("port_prop_sensor"="Port Propulsion Sensor")
+	sensors = list("port_prop_sensor"="Port  Propulsion  Sensor")
```
Comment (DreamySkrell) [inline maps/sccv_horizon/sccv_horizon-1_deck_1.dmm:12388]: ?

Comment (Snowy1237) [reply maps/sccv_horizon/sccv_horizon-1_deck_1.dmm]: Issue solved.

**Review by DreamySkrell — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**